www_src/pages/project/project.jsx

```diff
@@ -437,6 +437,30 @@ var Project = React.createClass({
     }
   },
 
+  setDestination: function () {
+    // var pageUrl = `/projects/${this.state.params.project}/pages/${this.state.routeData.pageID}`;
+
+    api({
+      method: 'patch',
+      uri: `/users/1/projects/${this.state.params.project}/pages/${this.state.routeData.pageID}/elements/${this.state.routeData.elementID}`,
+      json: {
+        attributes: {
```

Comment:
The reason I set it up the way it is is because if I merge the incoming JSON with the existing JSON in the db, there's no simple way to delete existing attributes besides setting them to `''` or `undefined`